catchpoint-mixin/targets.libsonnet

```diff
@@ -105,15 +106,235 @@ local utils = commonlib.utils {
+        'sum by (test_name) (catchpoint_html_content_type{%(testNameSelector)s})' % vars
+      )
+      + prometheusQuery.withLegendFormat('html'),
+
+    cssLoadedBySize:
+      prometheusQuery.new(
+        '${' + vars.datasources.prometheus.name + '}',
+        'sum by (test_name) (catchpoint_css_content_type{%(testNameSelector)s})' % vars
+      )
+      + prometheusQuery.withLegendFormat('css'),
+
+    fontLoadedBySize:
+      prometheusQuery.new(
+        '${' + vars.datasources.prometheus.name + '}',
+        'sum by (test_name) (catchpoint_font_content_type{%(testNameSelector)s})' % vars
+      )
+      + prometheusQuery.withLegendFormat('font'),
+
+    xmlLoadedBySize:
+      prometheusQuery.new(
+        '${' + vars.datasources.prometheus.name + '}',
+        'sum by (test_name) (catchpoint_xml_content_type{%(testNameSelector)s})' % vars
+      )
+      + prometheusQuery.withLegendFormat('xml'),
+
+    mediaLoadedBySize:
+      prometheusQuery.new(
+        '${' + vars.datasources.prometheus.name + '}',
+        'sum by (test_name) (catchpoint_media_content_type{%(testNameSelector)s})' % vars
+      )
+      + prometheusQuery.withLegendFormat('media'),
+
+    imageLoadedByType:
+      prometheusQuery.new(
+        '${' + vars.datasources.prometheus.name + '}',
+        'sum by (test_name) (catchpoint_image_count{%(testNameSelector)s})' % vars
+      )
+      + prometheusQuery.withLegendFormat('image'),
+
+    scriptLoadedByType:
+      prometheusQuery.new(
+        '${' + vars.datasources.prometheus.name + '}',
+        'sum by (test_name) (catchpoint_script_count{%(testNameSelector)s})' % vars
+      )
+      + prometheusQuery.withLegendFormat('script'),
+
+    htmlLoadedByType:
+      prometheusQuery.new(
+        '${' + vars.datasources.prometheus.name + '}',
+        'sum by (test_name) (catchpoint_html_count{%(testNameSelector)s})' % vars
+      )
+      + prometheusQuery.withLegendFormat('html'),
+
+    cssLoadedByType:
+      prometheusQuery.new(
+        '${' + vars.datasources.prometheus.name + '}',
+        'sum by (test_name) (catchpoint_css_count{%(testNameSelector)s})' % vars
+      )
+      + prometheusQuery.withLegendFormat('css'),
+
+    fontLoadedByType:
+      prometheusQuery.new(
+        '${' + vars.datasources.prometheus.name + '}',
+        'sum by (test_name) (catchpoint_font_count{%(testNameSelector)s})' % vars
+      )
+      + prometheusQuery.withLegendFormat('font'),
+
+    xmlLoadedByType:
+      prometheusQuery.new(
+        '${' + vars.datasources.prometheus.name + '}',
+        'sum by (test_name) (catchpoint_xml_count{%(testNameSelector)s})' % vars
+      )
+      + prometheusQuery.withLegendFormat('xml'),
+
+    mediaLoadedByType:
+      prometheusQuery.new(
+        '${' + vars.datasources.prometheus.name + '}',
+        'sum by (test_name) (catchpoint_media_count{%(testNameSelector)s})' % vars
+      )
+      + prometheusQuery.withLegendFormat('media'),
+
+    errors:
```

Comment:
any errors is not used here. All the errors are for this panel.

Reply:
Also, this displays true/false values indicating if there are errors or not. I am not sure the best way to graph this. Maybe on 0, do not show the metric and set a max to 1 value. Curious to know your thoughts on this.